packages/cdktf-cli/src/bin/cmds/helper/terraform-cloud-client.ts

```diff
@@ -117,9 +160,14 @@ async function post(url: string, token: string, data: string) {
   });
 }
 
+async function endpointUrl(tfeHostname: string, path: string) {
+  const serviceDiscovery = await discoverService(tfeHostname);
+  return `https://${tfeHostname}${serviceDiscovery["tfe.v2"]}${path}`;
```

Comment:
Also, i see inconsistencies between my experience with artifactory, and terraform docs, whether the service URLs end with a `/` or not. It would be safer to use a library to combine paths